chat-plugins/dice.js

```diff
@@ -1,174 +0,0 @@
-/*Dice Game plugin by SilverTactic (Silveee)*/
```

Comment:
Theres no reason this cant be in its own file, it just needs to have the currency handlers edited to use the new system.